### ISSUE TITLE
Unable to Launch NorthstarMP from Main Menu + Authentication error.

### ISSUE THREAD

**Issue by ColonelJRico** [R2Northstar/Northstar#95]
![image](https://user-images.githubusercontent.com/96916702/147828156-f3e257b2-5682-44eb-9ffc-5d009baeb715.png)

I'm unable to launch Northstar MP, able to access lobby only after launching Campaign and wait around 2 minutes however no servers are available and any UI interaction boots me back to Main screen.

Authentication Failed dialogue apppears after a few minutes when doing nothing on Main screen.

**Comment by ColonelJRico** [comment]
can send consol stuff if that helps

**Comment by MiloAkerman** [comment]
Duplicate of #78
